src/modules/postResCom.js

```diff
@@ -0,0 +1,31 @@
+import * as commentDetails from './commentInterface.js';
+
+const commentPost = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/HrIKPRrYjrxS00NlIVCD/comments/';
+
+const postCommentData = (imgId) => {
+  let count = 0;
+  const theForm = commentDetails.commentForm;
+  theForm.addEventListener('submit', async (ev) => {
+    ev.preventDefault();
+    count += 1;
+    if (count === imgId + 1) {
+      //    const itemid = String(imgId);
```

Comment:
- [ ] Good job removing previous comments but this single line will reproduce the same effects the previewer mentioned, making your code messy. Kindly remove it pls :pray: